tests/semanticdb/expect/Local.expect.scala

```diff
@@ -2,7 +2,7 @@ package example
 
 class Local/*<-example::Local#*/ {
   def a/*<-example::Local#a().*/() = {
-    def id/*<-local0*/[A/*<-local1*/](a/*<-local2*/: A/*->local1*/): A/*->local1*/ = a/*->local2*/
-    id/*->local0*/(1)
+    def id/*<-local2*/[A/*<-local0*/](a/*<-local1*/: A/*->local0*/): A/*->local0*/ = a/*->local1*/
```

Comment:
so the Scala 2 semanticdb implementation matches the old version exactly, whereas here the locals are in the opposite order. (I will look to see if there is a way to avoid reversing the traversal order in `ExtractSemanticDB`)

Reply:
As you may know, this is because of the change in traversing order. And I think this isn't a big deal for SemanticDB consumers as long as the same (compiler's) symbol has the same (semanticdb's) symbol.